tools/downloader/topologies_test.py

```diff
@@ -0,0 +1,103 @@
+        model_name = 'bert-large-uncased-whole-word-masking-squad-int8-0001'
+        face_detection = omz.Model(model_name, precisions=p)
+        xml_path, bin_path = face_detection.get_ir(precision=pr)
+        
+        self.assertTrue(os.path.exists(xml_path))
+        self.assertTrue(os.path.exists(bin_path))
+
+        self.assertEqual(os.path.basename(xml_path), model_name + '.xml')
+        self.assertEqual(os.path.basename(bin_path), model_name + '.bin')
+
+        self.assertEqual(os.path.basename(os.path.dirname(xml_path)), pr if pr else p[0])
+        self.assertEqual(os.path.basename(os.path.dirname(bin_path)), pr if pr else p[0])
+
+    @parameterized.expand([[['FP16'], 'FP16'], [['FP32'], 'FP32'],
+                           [None, 'FP16'], [None, 'FP32'],
+                           [['FP16'], None], [['FP32'], None]])
+    def test_load_public(self, p, pr):
+        ssd = omz.Model('mobilenet-ssd', precisions=p)
+        xml_path, bin_path = ssd.get_ir(precision=pr)
+        
+        self.assertTrue(os.path.exists(xml_path))
+        self.assertTrue(os.path.exists(bin_path))
+
+        self.assertEqual(os.path.basename(xml_path), 'mobilenet-ssd.xml')
+        self.assertEqual(os.path.basename(bin_path), 'mobilenet-ssd.bin')
+
+        self.assertEqual(os.path.basename(os.path.dirname(xml_path)), pr if pr else p[0])
+        self.assertEqual(os.path.basename(os.path.dirname(bin_path)), pr if pr else p[0])
+
+    def test_result_model(self):
+        ssd = omz.Model('mobilenet-ssd')
+        image_path = os.path.join(os.path.split(os.path.dirname(os.path.realpath(__file__)))[0],
+                                  'accuracy_checker', 'data', 'test_data', '1.jpg')
+
+        ie = IECore()
+        xml_path, bin_path = ssd.get_ir(precision='FP32')
+        net = ie.read_network(xml_path, bin_path)
+        exec_net = ie.load_network(net, 'CPU')
+
+        image = cv.imread(image_path)
+
+        image = cv.resize(image, (300, 300))
+        image = np.expand_dims(image, axis=0)
+        image = image.transpose((0, 3, 1, 2))
+
+        input_blob = 'data'
+        out_blob = next(iter(net.outputs))
+
+        predictions = exec_net.infer(inputs={input_blob: image})
+        predictions = predictions[out_blob]
+
+        for detection in predictions.reshape(-1, 7):
+            image_id = detection[0]
+            if image_id < 0:
+                break
+
+            confidence = detection[2]
+            label = int(detection[1])
+            xmin = detection[3]
+            ymin = detection[4]
+            xmax = detection[5]
+            ymax = detection[6]
+
+        delta = 1e-05
+        self.assertEqual(label, 7)
+        self.assertAlmostEqual(confidence, 0.9891584, delta=delta)
+        self.assertAlmostEqual(xmin, 0.34685153, delta=delta)
+        self.assertAlmostEqual(ymin, 0.2038618, delta=delta)
+        self.assertAlmostEqual(xmax, 0.5851372, delta=delta)
+        self.assertAlmostEqual(ymax, 0.87824464, delta=delta)
```

Comment:
I don't think we need this. It tests the _model_ rather than the API, and we don't need tests for models - we already have accuracy validation. The problem with it is that it makes the tests high-maintenance - if this model is removed, we'll have to redo the test to match another model. Also, the results may diverge from the reference values due to changes in IE or CPU features supported on the machine that runs them.